tests/src/test/scala/apigw/healthtests/ApiGwEndToEndTests.scala

```diff
@@ -142,6 +142,72 @@ class ApiGwEndToEndTests
         }
     }
 
+    it should "list api-experimental alphabetically by Base/Rel/Verb and Action name" in withAssetCleaner(wskprops) {
+    (wp, assetHelper) =>
+
+        val actionName1 = "actionName1"
+        val actionName2 = "actionName2"
+        val actionName3 = "actionName3"
+        val base1 = "/BaseTestPath1"
+        val base2 = "/BaseTestPath2"
+        val base3 = "/BaseTestPath3"
+
+        try {
+            // Create Actions for apiexperimentals
+            val file = TestUtils.getTestActionFilename(s"echo-web-http.js")
+            assetHelper.withCleaner(wsk.action, actionName1) {
```

Comment:
As mentioned in other comments, this can be handled inside a loop to avoid having copied code.